packages/ui/src/components/FloatingActionButton.tsx

```diff
@@ -32,3 +32,5 @@ export default function FloatingActionButton({
     </Button>
   );
 }
+
+export { FloatingActionButton };
```

Comment:
Why is this exported twice? Should we just switch to this export statement and remove the default export?

Reply:
Sure, wasn't sure what the convention was. Will switch to named